server/controllers/request.js

```diff
@@ -0,0 +1,64 @@
+
+   
+const Requset = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /request/
+// @desc Get this user all request 
+// @access Private
+exports.getAllRequest = asyncHandler(async (req, res, next) => {
+    const userId  = req.user.id
+    const requests = await Request.find({userId:userId})
+
+    res.status(200).json({success:{requests}})
```

Comment:
Not sure why we are squishing everything together. Can we try going forward to space things a part for more readability? Thanks!

Also, `success` property name doesn't quite make sense. We can just do `{ requests }` and that will be fine.